packages/dev/core/src/PostProcesses/RenderPipeline/Pipelines/taaMaterialManager.ts

```diff
@@ -0,0 +1,174 @@
+import type { Material } from "core/Materials/material";
+import { MaterialDefines } from "core/Materials/materialDefines";
+import { MaterialPluginBase } from "core/Materials/materialPluginBase";
+import { RegisterMaterialPlugin, UnregisterMaterialPlugin } from "core/Materials/materialPluginManager";
+import type { UniformBuffer } from "core/Materials/uniformBuffer";
+import type { Scene } from "core/scene";
+import { Vector2 } from "core/Maths/math.vector";
+import type { Nullable } from "core/types";
+import { ShaderLanguage } from "core/Materials";
```

Comment:
the path should be the full path and not to a folder which is the root cause of the build issue.